README.md

```diff
@@ -49,6 +49,7 @@ Attempts to download a video from the given url. Returns a [readable stream](htt
   ```
 * `format` - Primarily used to download specific video or audio streams. This can be a specific `format` object returned from `getInfo`.
   * Supplying this option will ignore the `filter` and `quality` options since the format is explicitly provided.
+* `dlChunkSize` - The size of the download chunk in bytes, this is used when the chosen format is video only or audio only, the download in this case is separated into multiple chunks to avoid throttling. Defaults to 10MB.
```

Comment:
hmm, i think this should be moved down after `highWaterMark`. most used options should go near the top. and `dlChunkSize` is not something users will be changing unless they experience more throttling issues.